CONTRIBUTING.md

```diff
@@ -829,6 +830,21 @@ class Event(models.Model):
   a_field(..., default=lambda self: self._default_get())
   ```
 
+### Exceptions
+  The `pass` into block except is not a good practice!
+
+  By including the `pass` we assume that our algorithm can continue to function after the exception occurred
+
+  If you really need to use the `pass` consider logging that exception
+
+  ```python
+    try:
+        sentences
+    except:
+        _logger.debug('Pass')
```

Comment:
Wow ok sorry, this is totally just something with Github display & not something you did (it's a lowercase l looking at the code in IDE). Just so you know I'm not crazy though, this looks like some sort of uppercase unicode L derivate:

![image](https://cloud.githubusercontent.com/assets/7397578/22032574/b80a95f8-dc99-11e6-85a4-7a5dd4d71fd0.png)